<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportIncomingDirectionsSetupDialog</class>
 <widget class="QDialog" name="ReportIncomingDirectionsSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>Исполнитель</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QComboBox" name="cmbActionType"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblActionType">
     <property name="text">
      <string>Тип направления</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="CActionStatusComboBox" name="cmbActionStatus"/>
   </item>
   <item row="5" column="2" colspan="3">
    <widget class="COrgIsMedicalComboBox" name="cmbOrganisation"/>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="CPersonComboBox" name="cmbPerson"/>
   </item>
   <item row="0" column="4">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>101</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>95</width>
       <height>0</height>
      </size>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="lblOrganisation">
     <property name="text">
      <string>Направитель</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrganisation</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>101</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>95</width>
       <height>0</height>
      </size>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QTimeEdit" name="edtBegTime"/>
   </item>
   <item row="1" column="3">
    <widget class="QTimeEdit" name="edtEndTime"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lblActionStatus">
     <property name="text">
      <string>Состояние</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2" colspan="3">
    <widget class="QCheckBox" name="chkDetalLPU">
     <property name="text">
      <string>Детализировать по Целевым ЛПУ</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgIsMedicalComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CActionStatusComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">Events/ActionStatus.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtBegTime</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>edtEndTime</tabstop>
  <tabstop>cmbActionType</tabstop>
  <tabstop>cmbActionStatus</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>cmbOrganisation</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportIncomingDirectionsSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportIncomingDirectionsSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
